modules/ROOT/pages/create-endpoint.adoc

```diff
@@ -21,8 +21,8 @@ Ownership of facing endpoints (receive or send) is automatically assigned based
 * *Endpoints: Receive from partners*
 * *Endpoints: Send to partners*
 * *Endpoints: Target at <host-name>*
-* *Endpoints: Source at <host-name*
-. <<configure-endpoint,Configure the endpoint>> according the type of endpoint you are creating and the protocol you selected. 
+* *Endpoints: Source at <host-name>*
+. <<configure-endpoint,Configure the endpoint>> according the type of endpoint you are creating and the protocol you selected.
 . Click *Save*.
 
 == Creating Endpoints From the Message Flow
```

Comment:
Also, create from the message flow or the Message Flow wizard?
(ditto "using" suggestion)

Reply:
So, I cannot edit in text that is hidden/shaded blue (Lines 29 through 34 below), which indicates text that has not been manipulated in quite some time. Our group some time ago talked about identifying "pink," "green," and "blue" texts for edit, but I can't remember what the writers decided to do with the "blue" texts: ask Valkyrie?

Reply:
Also, this hidden text could use some work: what usage, do we mean send vs. receive?
You assign the owner by "making the appropriate selection" where?
Why bother having this topic at all if we just send the audience to other topics?

Reply:
Done for the first 4.

For the last one, I removed the ownership line. There are topics about creating endpoints from message flows and they are linked below. I feel like the inline links should remain because the details on this are actually explained elsewhere and not in these topics. There are two ways to create endpoints - manually and from the message flow. I added "Wizard" to the end of the heading and reworded the paragraph.